mitosheet/src/components/taskpanes/ControlPanel/FilterAndSortTab/filter/utils.tsx

```diff
@@ -1,40 +1,42 @@
 // Copyright (c) Mito
 
-import { BooleanFilterCondition, ColumnMitoType, DatetimeFilterCondition, FilterType, NumberFilterCondition, StringFilterCondition, FilterGroupType } from '../../../../../types';
+import { FilterType, FilterGroupType } from '../../../../../types';
+import { isBoolDtype, isDatetimeDtype, isNumberDtype, isStringDtype } from '../../../../../utils/dtypes';
 import { CONDITIONS_WITH_NO_INPUT } from './filterConditions';
 import { isFilterGroup } from './filterTypes';
 
+
 /*
     Returns the correct empty filter data object for the given columnMitoType
 */
-export function getEmptyFilterData(columnMitoType: ColumnMitoType): FilterType {
-    switch (columnMitoType) {
-        case ColumnMitoType.BOOLEAN_SERIES: return {
-            type: ColumnMitoType.BOOLEAN_SERIES,
-            condition: BooleanFilterCondition.IS_TRUE,
+export function getEmptyFilterData(columnDtype: string): FilterType {
+
+    if (isBoolDtype(columnDtype)) {
+        return {
+            condition: 'boolean_is_true',
             value: ''
         }
-        case ColumnMitoType.STRING_SERIES: return {
-            type: ColumnMitoType.STRING_SERIES,
-            condition: StringFilterCondition.CONTAINS,
+    } else if (isStringDtype(columnDtype)) {
+        return {
+            condition: 'contains',
             value: ''
         }
-        case ColumnMitoType.NUMBER_SERIES: return {
-            type: ColumnMitoType.NUMBER_SERIES,
-            condition: NumberFilterCondition.GREATER,
+    } else if (isNumberDtype(columnDtype)) {
+        return {
+            condition: 'greater',
```

Comment:
It seems like this is typed less strongly than before. Because the 'greater' condition exists on numbers and datetimes. Note to self: look into this more, where are types being maintained?